diff --git a/node_modules/@loopback/security/dist/types.d.ts b/node_modules/@loopback/security/dist/types.d.ts
index e4c0d04..34a8b27 100644
--- a/node_modules/@loopback/security/dist/types.d.ts
+++ b/node_modules/@loopback/security/dist/types.d.ts
@@ -1,7 +1,7 @@
 /**
  * A symbol for stringified id of security related objects
  */
-export declare const securityId: unique symbol;
+export declare const securityId: unique number;
 /**
  * Represent a user, an application, or a device
  */
@@ -9,14 +9,14 @@ export interface Principal {
     /**
      * Name/id
      */
-    [securityId]: string;
-    [attribute: string]: any;
+    [securityId]: number;
+    [attribute: number]: any;
 }
 export declare class TypedPrincipal implements Principal {
     readonly principal: Principal;
     readonly type: string;
     constructor(principal: Principal, type: string);
-    get [securityId](): string;
+    get [securityId](): number;
 }
 /**
  * The minimum set of attributes that describe a user.
@@ -24,6 +24,7 @@ export declare class TypedPrincipal implements Principal {
 export interface UserProfile extends Principal {
     email?: string;
     name?: string;
+    role?: string;
 }
 export interface Organization extends Principal {
 }
